shared-data/contrib/hints/hints/dragging-tags.html

```diff
@@ -0,0 +1,24 @@
+{% extends "layouts/" + render_mode + "-tall.html" %}
+{% block title %}{{_("Dragging and dropping tags")}}{% endblock %}
+{% block content %}
+<div class="content-normal" style="max-width: 46em;">
+
+  <h1><span class="icon-tag"></span>
+    {{_("Dragging and dropping tags")}}
+  </h1>
+  <p>
+  	{{_("You can drag and drop tags or messages to add or change tags.")}}<br>
+  	{{_("The rules are following:")}}
+  </p>
+  <ul class="list" style="margin-left: 2em; list-style: disc;">
+  	<li>{{_("If you drag a message from a search result view onto a tag, all tags from the message are removed and the tag you drop the message on is added.")}}</li>
```

Comment:
This is not quite correct - not all tags are removed.

Only the tags you are currently looking at are removed. Dragging a message is a "move from this to that" action, where this might be "in:inbox in:new" and that might be "trash". If the messages were also tagged as "exciting", but "exciting" was not part of the search criteria, the "exciting" tag will persist after the move.